Library/EntityGateway/LocalPersistence/Gateways/LocalPersistenceBooksGateway.swift

```diff
@@ -82,11 +82,10 @@ class CoreDataBooksGateway: LocalPersistenceBooksGateway {
 		
 		do {
 			try viewContext.save()
-			completionHandler(.success())
+            completionHandler(.success(()))
```

Comment:
I think instead of using .success(()) we should change the definition of of Result<ApiResponse<VoidResponse>>